QuizBowlDiscordScoreTracker/Program.cs

```diff
@@ -19,14 +21,14 @@ public static class Program
         private const int configFileReloadDelayMs = 30 * 1000;
 
         // Following the example from https://dsharpplus.emzi0767.com/articles/first_bot.html
-        public static void Main()
+        public static void Main(string[] args)
```

Comment:
Do we still need to accept args?

Reply:
yeah so we can pass it to `Host.CreateDefaultBuilder` otherwise ASP.NET doesn't pick up config from the CLI (useful since I can't get envvars working with `dotnet`...)